src/ui/server/helpers/convert-radio-response-boolean/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * converts response to boolean or returns an empty string if does not exist
+ * @param response from body which is string
+ * @returns boolean or empty string
+ */
+const convertRadioButtonResponse = (response: string) => {
+  if (response) {
+    // if response and if response is string true then return true, else false
+    return response === 'true' ? true : false;
+  }
+
+  return '';
+};
+
```

Comment:
We already have a helper function that sanitises/transforms such fields from form data, can we use `sanitiseData` please

Reply:
done